src/admin-views/settings/sidebars/default-sidebar.php

```diff
@@ -30,20 +31,23 @@
 );
 
 $sidebar = new Settings_Sidebar();
-$sidebar->set_header_image(
-	new Image(
-		tribe_resource_url( 'images/settings_illustration.jpg', false, null, Tribe__Events__Main::instance() ),
-		new Attributes(
-			[
-				'alt'  => '',
-				'role' => 'presentation',
-			]
+
+$hero_section = ( new Settings_Sidebar_Section() );
+$hero_section->header_image(
+		new Image(
+			tribe_resource_url( 'images/settings_illustration.jpg', false, null, Tribe__Events__Main::instance() ),
+			new Attributes(
+				[
+					'alt'  => '',
+					'role' => 'presentation',
+				]
+			)
 		)
 	)
-);
+	->set_title( new Heading( __( 'Finding and extending your calendar', 'tribe-common' ), 2, new Classes( 'tec-settings-form__sidebar-header' ) ) );
```

Comment:
Incorrect text domain
```suggestion
	->set_title( new Heading( __( 'Finding and extending your calendar', 'the-events-calendar' ), 2, new Classes( 'tec-settings-form__sidebar-header' ) ) );
```